tests/unit/mixins/tooltip-for-dropdown-test.js

```diff
@@ -10,3 +11,17 @@ test('it works', function(assert) {
   let subject = TooltipForDropdownObject.create();
   assert.ok(subject);
 });
+
+test('mouseEnter dropdownOpen is true', function(assert) {
+  let TooltipForDropdownObject = EmberObject.extend(TooltipForDropdownMixin);
```

Comment:
So when I started to expect I was forcing these to pass, I deleted line 16 and 17 and the test still passed. I'm assuming it's because I'm setting `dropdownOpen` to true above the assert, so it's not actually running the tooltip. However, at the same time, I'm pretty sure I'm pulling dropdown from our `TooltipforDropdownObject` so a part of me isn't sure how else I'd approach this.